add definition of junkie (#266)
tatianamac/selfdefined: 11ty/definitions/junkie.md

---
title: junkie

> slug: 'junkie'

Reviewer: ```suggestion
slug: junkie
```

speech: noun
defined: true
flag:
  level: avoid

>   text: 'ableist language'

Reviewer: ```suggestion
  text: 'ableist language'
  excerpt: a drug addict. a person who suffers from a complex disease of the brain and body that involves compulsive use of one or more substances despite serious health and social consequences.
```

> alt_words:

Reviewer: ```suggestion
excerpt: a drug addict. a person who suffers from a complex disease of the brain and body that involves compulsive use of one or more substances despite serious health and social consequences.
```

> alt_words:
>   - fanatic
>   - enthusiast
>   - lover
>   - fiend
>   - buff
>   - geek

Reviewer: ```suggestion
alt_words:
  - aficionado
  - buff
  - fanatic
  - enthusiast
  - fiend
  - geek
  - lover
```

---

> a drug addict. a person with a compulsive habit or obsessive dependency on something.

Reviewer 2: ```suggestion
a drug addict. a person who suffers from a complex disease of the brain and body that involves compulsive use of one or more substances despite serious health and social consequences.
```

Reviewer 2: I've updated the definition of an addict to reflect the latest terminology that focuses on addiction as a disease.

> a drug addict. a person who suffers from a complex disease of the brain and body that involves compulsive use of one or more substances despite serious health and social consequences.

Reviewer: ```suggestion
a drug addict; a person who suffers from a complex disease of the brain and body that involves compulsive use of one or more substances despite serious health and social consequences
```

## Issues

> Junkie is commonly used as a term for someone whose enthusiasm or zeal for something is extreme or beyond normal limits. However, this diminishes the seriousness of drug addiction and substance use disorder.

Reviewer 2: ```suggestion
Junkie is commonly used as a term for someone whose enthusiasm or zeal for something may be considered extreme or beyond normal interest by others. However, this diminishes the seriousness of drug addiction and substance use disorder.
```

Reviewer 2: removing "is" as that states fact where this really falls under opinion
